### ISSUE TITLE
Ubuntu 18.04 Bionic Support round #2

### ISSUE THREAD

**Issue by DigiAngel** [sans-dfir/sift#343]
Stale bot auto-closed.  Any idea when this will be supported?  Thank you.

**Comment by ekristen** [comment]
It's in the works, but no ETA. Hoping within the next couple of months.

**Comment by DigiAngel** [comment]
Brilliant thanks.

**Comment by joachimmetz** [comment]
@ekristen what is blocking here? Anything others could help with?

**Comment by ekristen** [comment]
> @ekristen what is blocking here? Anything others could help with?

I don't have a good answer to the first half right now. At a high level, it involves ensuring all the packages we need can be installed, including python packages. From there since Bionic left Unity and went back to Gnome, we have to ensure all our userland desktop changes get modified to work.

I've been already keeping notes on what is and is not available. I'll most likely open a series of issues soon as it pertains to 18.04 to shed light on the timeline and what work is left.

> For context we are considering dropping Xenial (16.04) support, so we can focus on Bionic and later to include Timesketch.

What does that mean? No more updates or removal of the packages from the GIFT PPA for Xenial like Trusty?

**Comment by joachimmetz** [comment]
> What does that mean? No more updates or removal of the packages from the GIFT PPA for Xenial like Trusty?

Trusty is already gone, seeing end-of-life in April, and increasing maintenance burden. I also don't think keeping old versions around is good practice, seeing the pace that things are changing.

Xenial is still there, it a heads up that we'll stop providing updates at some point and remove them all together at some point. It would be good for us to know what SIFT its plans are for Bionic, Xenial (and Trusty).

**Comment by ekristen** [comment]
SIFT is more or less following the LTS timeline for Ubuntu for support (minus a year maybe). Realistically the class USB drives need to be based on Bionic 18.04 for at least 6-12 months before we can even consider no longer supporting 16.04. 

Bionic - ASAP
Xenial - As long as the LTS is supported or the class USBs have been updated to Bionic for 6-12 months
Trusty - No longer supported.

**Comment by ekristen** [comment]
Closing in favor of #348

**Comment by joachimmetz** [comment]
> Xenial - As long as the LTS is supported or the class USBs have been updated to Bionic for 6-12 months

@ekristen are you tracking this in an issue? So I can link to that once we concluded what to do with Xenial on our side

**Comment by ekristen** [comment]
https://github.com/sans-dfir/sift/issues/348 and https://github.com/sans-dfir/sift/issues/349

**Comment by joachimmetz** [comment]
thx
